src/cmds/import.ts

```diff
@@ -12,7 +12,7 @@ export const desc = 'Kick off API powered import';
 export const builder = {
   file: {
     required: false,
-    default: 'import-projects.json',
+    default: false,
```

Comment:
this is a breaking change to users that used the original version by setting only the vars as now `--file` will always be set, lets discuss the bug you are experiencing to find a non braking fix